configs/missiles/flamer.model.cfg

```diff
@@ -1,10 +1,6 @@
 // display
 sound              models/weapons/flamer/fireloop
 
-dlight             200
```

Comment:
So basically this controls the dlight on the individual fire thingies that come out of the flamer. I think even wtih zero intensity, it does give a tiny amount of light. Whether that should be changed is debatable (personally, I think a more lit flamer looks cool).

Reply:
IIRC (because this PR is 2 months old), `dlight` means `dlightRange`, so intensity would still be 0? Unless you noticed something